parla/task_runtime.py

```diff
@@ -1824,7 +1881,7 @@ def _construct_datamove_task(self, target_data, compute_task: ComputeTask, opera
     def _map_tasks(self):
         # The first loop iterates a spawned task queue
         # and constructs a mapped task subgrpah.
-        #logger.debug("[Scheduler] Map Phase")
+        # logger.debug("[Scheduler] Map Phase")
```

Comment:
the below while loop (line 1829) needs to be break when there is no task on the queue, doesn't it?

Reply:
This is checked at L1888 because `_dequeue_spawned_task` returns None if it hits an IndexError.